src/app/complex/complex-details/complex-expression/cell-lines/cell-lines.component.html

```diff
@@ -0,0 +1,33 @@
+<div class="row">
+  @if (cellLines() && cellLines().length > 0) {
+    <div class="columns medium-12 no-pad-left">
+      <h3>Cell Lines ({{cellLines().length}})</h3>
+      <table class="hover">
+        <thead>
+        <th style="width: 30%">Identifier</th>
+        <th style="width: 70%">Name</th>
+        </thead>
+        <tbody>
+          @for (cellLine of cellLines(); track cellLine) {
+            <tr>
+              <td style="width: 30%">
+                @if (!!cellLine.searchURL) {
+                  <a href="{{cellLine.searchURL}}" target="_blank">
```

Comment:
Should use `[href]=...` instead of `{{..}}`. Also, sic e it's the only difference between the 2 ifs, can be refactored in a single block 
```
<a [href]="cellLine.searchURL || cellLineOntologySearchUrl(cellLine.identifier)" target="_blank"> {{celLine.identifier}} <i .../><a/>
```